src/styles.module.pcss

```diff
@@ -0,0 +1,211 @@
+.nft-tool {
+  margin: 1em auto;
+  display: flex;
+  flex-direction: row;
+  /* flex-direction: column; */
+
+  align-items: stretch;
+  padding: 20px 0;
+  gap: 30px;
+
+  &__form {
+    width: 100%;
+
+    &-param:not(:last-child) {
+      margin-bottom: 16px;
+    }
+
+    &-label {
+      display: block;
+      font-weight: bold;
+      font-size: 14px;
+      padding-bottom: 6px;
+    }
+
+    &-input {
+      width: 100%;
+      padding: 8px 12px;
+      border-radius: 4px;
+      border: 1px solid #EFF0F1;
```

Comment:
pls use css variables for all colours (at least). You can use them without posts plugin, since they supported natively